routing/message/handlers/passthrough/command_handler.go

```diff
@@ -34,10 +34,9 @@ const (
 	msgInvalidCloudCommand = "invalid cloud command"
 )
 
-// A simple command passthrough handler that forwards all cloud-to-device messages from Azure IoT Hub to local MQTT broker on a preconfigured topic.
 type commandHandler struct{}
 
-// CreateCommandHandler instantiates a new command handler that forward cloud-to-device messages to the local message broker using the given topic.
+// CreateCommandHandler instantiates a new command handler that forwards cloud-to-device messages to the local message broker using topics command//+/req/# and c//+/q/#.
```

Comment:
topics command//+/req/# and c//+/q/# - these two are not topics but filters. In general, I would suggest to rephrase it to: to the local message broker as Hono commands.